env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+envvars=( 'APP_URL' 'HMDA_API' 'KEYCLOAK_URL' )
+defaults=( 'http://192.168.99.100' 'https://192.168.99.100:4443/hmda' 'https://192.168.99.100:8443/auth/realms/hmda' )
```

Comment:
I know I brought up adding defaults when we talked about this, but do we really want them?  What will the experience be the defaults are applied, but they're wrong?  Will it be apparent to the dev what's going wrong?  Also, are you planning on setting them in `docker-compose.yml`?  Seems like you should just to give the hint of what needs to be changed when you're not using the default...and if it is going to be there, why even have the default?

In comparison, [`keycloak/docker-entrypoint.sh`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/docker-entrypoint.sh) exits on almost all of the envvars if they're not present.